pkg/test/common/utils.go

```diff
@@ -177,3 +179,31 @@ func CustomRedirectPolicy(noOfRedirect int) resty.RedirectPolicy {
 		return nil
 	})
 }
+
+// Generates a random string with length 10 from lower case & upper case characters.
+func GenerateRandomString() string {
+	//nolint: gosec
+	seededRand := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
Log this seed so that if there is a test failure that I need to reproduce, I can manually set it to that seed value.